README.md

```diff
@@ -1,23 +1,22 @@
-# Run-on-Slack: Translator App (powered by DeepL API)
+# Deno Message Translator App (powered by DeepL API)
 
 This app contains a sample TypeScript project for use on Slack's
 [next-generation hosted platform](https://api.slack.com/future). The project
-demonstrates how to run a simple message translator workflow in response to
-national flag based reactions. Admins can configure where to run this app using
-a link trigger based interactions to maintain the list of channels. Once the app
-is added to a channel, end-users can translate any messages into a different
-language just by adding a reaction such as 🇺🇸, 🇪🇸, 🇫🇷, 🇯🇵, and more! To learn
+demonstrates a simple message translator app. The app is added to designated channels by running a configurator workflow. Once added to specific channel(s), the app will translate any message there when a user adds a reaction to the message (ex: 🇺🇸, 🇪🇸, 🇫🇷, 🇯🇵, and more)!
+
+To learn
 the full list of the supported languages, head to
 [the DeepL API's document site](https://www.deepl.com/en/docs-api).
 
 **Guide Outline**:
 
+- [Supported Workflows](#supported-workflows)
 - [Setup](#setup)
   - [Create Your DeepL API Account](#create-your-deepl-api-account)
   - [Install the Slack CLI](#install-the-slack-cli)
   - [Clone the Template](#clone-the-template)
   - [Save Env Values](#save-env-values)
-- [Create a link trigger](#create-a-link-trigger-for-configuring-your-app)
+- [Create a Link Trigger](#create-a-link-trigger-for-configuring-your-app)
```

Comment:
Why did you revert this? You asked to consistently make these terms lower-cased at #1

Reply:
Ah yes, my apologies! I reverted it because I realized it was a title of the section, so it did make sense to capitalize it here. Sorry for the confusion!

Reply:
Ah, this makes sense! Thanks for clarifying this